src/gbinder_driver.c

```diff
@@ -447,9 +447,21 @@ gbinder_driver_handle_transaction(
             tx.code, tx.flags, &status);
         break;
     case GBINDER_LOCAL_TRANSACTION_SUPPORTED:
-        reply = gbinder_handler_transact(h, obj, req, tx.code, tx.flags,
-            &status);
-        break;
+        {
+            if(handler)
+            {
+                // this function will be called if we have none null handler
```

Comment:
And this looks looks like a C++ style comment. It also doesn't make sense - how can we get here with a NULL handler if it was checked (to be non-NULL) one line earlier?